icu4c/source/common/unicode/uset.h

```diff
@@ -1285,4 +1325,414 @@ U_CAPI UBool U_EXPORT2
 uset_getSerializedRange(const USerializedSet* set, int32_t rangeIndex,
                         UChar32* pStart, UChar32* pEnd);
 
-#endif
+#if U_SHOW_CPLUSPLUS_API
+#ifndef U_HIDE_DRAFT_API
+
+namespace U_HEADER_ONLY_NAMESPACE {
+
+// TODO: Naming: For now, I am working with UnicodeSetXyz class prefixes.
+// This might be a little confusing, since these classes actually operate on a USet,
+// the C wrapper around a C++ UnicodeSet.
+// I am thinking of renaming the type prefixes to just USetXyz.
+// Opinions?
```

Comment:
I think you can take this comment out now.

Reply:
whoops! thanks -- done